engine/src/test/java/com/redhat/cloud/notifications/EmailTemplatesRendererHelper.java

```diff
@@ -101,7 +102,7 @@ protected String generateAggregatedEmailBody(Map<String, Object> originalContext
             Arrays.stream(sections).filter(e -> !e.isBlank()).toList());
 
         Map<String, Object> mapData = Map.of("title", "Daily digest - Red Hat Enterprise Linux", "items", List.of(dailyDigestSection), "orgId", DEFAULT_ORG_ID);
-        TemplateDefinition globalDailyTemplateDefinition = new TemplateDefinition(IntegrationType.EMAIL_DAILY_DIGEST_BODY, null, null, null);
+        TemplateDefinition globalDailyTemplateDefinition = new TemplateDefinition(IntegrationType.EMAIL_DAILY_DIGEST_BODY, INTERNAL_DAILY_DIGEST_BUNDLE, null, null);
```

Comment:
We need to add a fake bundle to avoid this internal parent template to be considered as a daily digest template by default.